src/providers/twitch/api/Helix.cpp

```diff
@@ -1178,6 +1178,87 @@ void Helix::sendChatAnnouncement(
         .execute();
 }
 
+void Helix::addChannelVIP(
+    QString broadcasterID, QString userID, ResultCallback<> successCallback,
+    FailureCallback<HelixAddChannelVIPError, QString> failureCallback)
+{
+    using Error = HelixAddChannelVIPError;
+
+    QUrlQuery urlQuery;
+
+    urlQuery.addQueryItem("broadcaster_id", broadcasterID);
+    urlQuery.addQueryItem("user_id", userID);
+
+    this->makeRequest("channels/vips", urlQuery)
+        .type(NetworkRequestType::Post)
+        .onSuccess([successCallback, failureCallback](auto result) -> Outcome {
+            if (result.status() != 204)
+            {
+                qCWarning(chatterinoTwitch)
+                    << "Success result for deleting chat messages was"
```

Comment:
```suggestion
                    << "Success result for adding a VIP was"
```

Reply:
Yes - it's still a draft, this is only up to help with diffing some stuff